lib/create-organization-api.js

```diff
@@ -77,8 +79,53 @@ export default function createOrganizationApi ({
       .then(response => wrapOrganizationMembershipCollection(http, response.data), errorHandler)
   }
 
+  /**
+   * Gets an Organization Space Membership
+   * @memberof ContentfulOrganizationAPI
+   * @param {string} id - Organiztion Space Membership ID
+   * @return {Promise<OrganizationSpaceMembership.OrganizationSpaceMembership>} Promise for a Organization Space Membership
+   * @example
+   * const contentful = require('contentful-management')
+   *
+   * client.getOrganizations()
+   * .then((organization) => organization.getOrganizationSpaceMembership('organizationSpaceMembership_id'))
+   * .then((organizationMembership) => console.log(organizationMembership))
+   * .catch(console.error)
+   */
+  function getOrganizationSpaceMembership (id) {
+    return http.get('space_memberships/' + id, {
+      baseURL,
+      headers
+    })
+      .then(response => wrapOrganizationSpaceMembership(http, response.data, orgId), errorHandler)
+  }
+
+  /**
+   * Gets a collection of Organization Space Memberships
+   * @memberof ContentfulOrganizationAPI
+   * @param {object=} query - Object with search parameters. Check the <a href="https://www.contentful.com/developers/docs/javascript/tutorials/using-js-cda-sdk/#retrieving-entries-with-search-parameters">JS SDK tutorial</a> and the <a href="https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/search-parameters">REST API reference</a> for more details.
+   * @return {Promise<OrganizationSpaceMembership.OrganizationSpaceMembershipCollection>} Promise for a collection of Organization Memberships
```

Comment:
small docs update: "a collection of Organization Space Memberships"

Reply:
on my way! 🚐

Reply:
That would be misleading.
I would just declare `{Promise<SpaceMembershipCollection>}` and say "Collection of Space Memberships across all spaces in the organization